test/Header/SetCookieTest.php

```diff
@@ -431,8 +432,8 @@ public function testPreventsCRLFAttackViaFromString()
      */
     public function testPreventsCRLFAttackViaConstructor()
     {
-        $this->setExpectedException('Zend\Http\Header\Exception\InvalidArgumentException');
         $header = new SetCookie("leo_auth_token", "example\r\n\r\nevilContent");
+        $this->assertTrue(HeaderValue::isValid($header->getFieldValue()));
```

Comment:
I preffer if you assertEquals against the __toString() output. Will be more clear to understand because the encoded characters will be visible while reading this code.